.github/workflows/eventlist.yml

```diff
@@ -194,17 +197,16 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04, windows-2019, macos-12]
-        go: [1.19.3]
     name: 'Test (${{ matrix.os }}, ${{ matrix.go }})'

```

Comment:
`${{ matrix.go }}` is not set anymore

Reply:
Ahh My bad. Fixed it
[Address missing review comments](https://github.com/ARM-software/CMSIS-View/pull/14/commits/d8b0aa11de4d1688de7f97c1bff7beb73562326f)